src/main/resources/swagger/api-docs.yaml

```diff
@@ -139,6 +147,45 @@ definitions:
         type: integer
         description: line number
 
+  Cluster:
+    description: ''
+    required:
+      - clusterId
+      - clusterName
+      - googleProject
+      - googleServiceAccount
+      - googleBucket
+      - operationName
+      - createdDate
+      - labels
+    properties:
+      clusterId:
+        type: string
+        description: Google's UUID for the cluster
```

Comment:
probs also good to check in w/bernick to see if we should be using ids known to the user as database PKs; may be a Security Thing